pkgs/applications/networking/instant-messengers/telegram/telegram-desktop/unwrapped.nix

```diff
@@ -98,6 +102,13 @@ stdenv.mkDerivation (finalAttrs: {
       boost
       ada
       (tdlib.override { tde2eOnly = true; })
+      # even though the last 3 dependencies are already in `kimageformats`,
+      # because of a logic error in the cmake files, in td 5.15.{1,2} it
+      # doesn't link when you don't add them explicitly
+      kimageformats
+      libavif
+      libheif
+      libjxl
```

Comment:
Any upstream issue on this?

Reply:
> they tend to be quick but I guess not this time

I sleep sometimes too 😁

> they are in this thread already, they might be able to clarify a bit better

Looks like I just have added the conditionals not in each needed place :see_no_evil:

Reply:
> but according to Github 5.15.1 got released slightly later than 5.15.1

I bet preston ran release scripts simultaneously and the one releasing 5.15.1 has ended later

Reply:
Yeah no worries, @nickcao they fixed it on trunk, so this could be removed next time.